src/amber/cli/commands/sentry_command.cr

```diff
@@ -0,0 +1,103 @@
+require "yaml"
+require "cli"
+
+require "sentry"
+
+module Sentry
```

Comment:
Is this meant to replace `watch`

Reply:
No, It's just a helper to avoid using https://github.com/TechMagister/sentry/tree/change_cli

Reply:
I moved this file and rename to more explicit name